client/app/screens/Loading.js

```diff
@@ -59,16 +60,29 @@ export default class LoadingScreen extends React.Component {
       .then(tokens => {
         if (tokens) {
           this.setState(
-            { hasToken: true, token: tokens['access_token'] },
+            {
+              hasToken: true,
+              token: tokens['access_token'],
+            },
             this._attemptNavigation
           );
         } else {
-          this.setState({ hasToken: false }, this._attemptNavigation);
```

Comment:
This file was a lot more readable with these `setState` calls on single lines IMO. Do you mind rolling this back?